binaries/setup-cloud-scoot/main.go

```diff
@@ -31,13 +38,13 @@ func main() {
 
 	workersCfg := &setup.WorkerConfig{Count: *workersFlag, RepoDir: *repoDir}
 	sched := map[string]setup.SchedulerStrategy{
-		"local.memory": setup.NewLocalMemory(workersCfg, builder, cmds),
-		"local.local":  setup.NewLocalLocal(workersCfg, builder, cmds),
+		"local.memory": setup.NewLocalMemory(workersCfg, builder, cmds, level),
```

Comment:
Can you move level into the WorkerConfig struct instead? Same for scheduler and apiserver.